database/schema.graphql

```diff
@@ -1389,7 +1389,7 @@ input ApplicationResponseTemplateQuestionIdFkeyTemplateElementCreateInput {
   visibilityCondition: JSON
```

Comment:
Huh, this file is in `.gitignore`. Do you know why it's showing up here?

Reply:
Not sure. I have deleted it now.